driver/src/main/java/org/neo4j/driver/internal/InternalExecutableQuery.java

```diff
@@ -107,4 +116,9 @@ public Map<String, Object> parameters() {
     public QueryConfig config() {
         return config;
     }
+
+    // For testing only
+    public AuthToken authToken() {
+        return authToken;
+    }
```

Comment:
Might worth to downgrade it to protected/package if possible.

Reply:
It is an internal class (module descriptor), but we might as well downgrade to package visibility here, thanks. 👍